styles.css

```diff
@@ -474,11 +451,100 @@ hr {
   margin-top: 20px;
 }
 
-.skills-container ul li {
+.contact-section {
+  width: 87%;
+  padding: 120px 0;
+  margin: auto;
+  display: flex;
+  flex-direction: column;
+}
+
+.contact-section .contact-head {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+  color: #172b4d;
+  align-self: center;
+}
+
+.contact-section .form {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: auto auto auto auto;
+  gap: 20px;
+  margin-top: 40px;
+  align-items: center;
```

Comment:
- [ ] You did well 😊 at placing the elements, but as stated in the requirements try to use the flexbox not the grid system to place the elements in the form 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Project requirements |
| --- |
|  ![image](https://user-images.githubusercontent.com/83360636/222165775-7bbdc565-eaa1-4713-b05c-a331713257f1.png)   |